src/cmdstan/arguments/arg_log_prob.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef CMDSTAN_ARGUMENTS_ARG_LOG_PROB_HPP
+#define CMDSTAN_ARGUMENTS_ARG_LOG_PROB_HPP
+
+#include <cmdstan/arguments/arg_log_prob_unconstrained_params.hpp>
+#include <cmdstan/arguments/arg_log_prob_constrained_params.hpp>
+#include <cmdstan/arguments/arg_log_prob_jacobian_adjust.hpp>
+#include <cmdstan/arguments/categorical_argument.hpp>
+
+namespace cmdstan {
+
+class arg_log_prob : public categorical_argument {
```

Comment:
Classes and constructors should get doc.

Reply:
I've added doc for the others, but I think it might be redundant to add doc here. The description covers what the method does, and the sub-arguments have doc for their purpose and type